### PR TITLE
custom: adding convenient entry points for custom --args subcommand

### PR DESCRIPTION
**Pull request by jbohren**

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -51,6 +51,10 @@
             'vcs-push = vcstool.commands.push:main',
             'vcs-remotes = vcstool.commands.remotes:main',
             'vcs-status = vcstool.commands.status:main',
+            'vcs-git = vcstool.commands.custom:git_main',
+            'vcs-hg = vcstool.commands.custom:hg_main',
+            'vcs-svn = vcstool.commands.custom:svn_main',
+            'vcs-bzr = vcstool.commands.custom:bzr_main',
```
Comment (dirk-thomas) [inline setup.py]: Please add those line in alphabetical order.

File: vcstool/commands/custom.py [hunk 2]
```diff
@@ -72,6 +72,17 @@ def main(args=None):
     output_results(results, hide_empty=args.hide_empty)
     return 0
 
+def git_main(args=None):
+    return main(['--git','--args']+(args if args else sys.argv[1:]))
```
Comment (dirk-thomas) [inline vcstool/commands/custom.py]: The condition is to unspecific. If the function gets invoked e.g. with an empty list it falls back to sys.argv. Please compare with `is not None`.

For readability can you also please move setting the args before the return:

    if args is None:
        args = sys.argv[1:]

Please add a space after the comma and around the plus operator (PEP8) - same below.

**Comment by dirk-thomas** [comment]
Please also add the corresponding scripts which makes it easier to use this package from source.

**Comment by jbohren** [comment]
Feedback issues resolved.

File: scripts/vcs-bzr [hunk 3]
```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/env python
+
+import sys
+from vcstool.commands.custom import bzr_main
+
+sys.exit(git_main() or 0)
```
Comment (dirk-thomas) [inline scripts/vcs-bzr]: Calling the wrong function - same below.

File: vcstool/commands/custom.py [hunk 4]
```diff
@@ -72,6 +72,25 @@ def main(args=None):
     output_results(results, hide_empty=args.hide_empty)
     return 0
 
+def bzr_main(args=None):
+    if args is None:
+        args = sys.argv[1:]
+    return main(['--bzr', '--args']+args)
```
Comment (dirk-thomas) [inline vcstool/commands/custom.py]: Please add spaces around the plus operator - same below.

**Comment by jbohren** [comment]
Dumb typos resolved.

**Comment by dirk-thomas** [comment]
Great - thank you!

**Pull request merged**